### PR TITLE
Draft post for "Improve Your Data Mesh with Real-Time Data Sharing"

### PR DESCRIPTION
**Pull request by james-vendia**
Final draft for your final review @caroll-vendia and @brian-vendia.

CC @francine-vendia  so she can observe how the final review process works

### CODE REVIEW

File: posts/2022-05-10-improve-your-data-mesh.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  - James Gimourginas
+---
+
+## Background
+
+[Domain Driven Design](https://www.oreilly.com/library/view/domain-driven-design-tackling/0321125215/) (DDD) is the basis for decomposing monolithic applications into collaborating (micro, in many cases) services.  The approach [gained in popularity](https://www.oreilly.com/radar/microservices-adoption-in-2020/) because it addressed some of the long-standing challenges of large, complex, monolithic operational applications.  Benefits of a microservices approach included:
+
+* **Agility and Autonomy** - Teams empowered to pursue business priorities with minimal cross-team touchpoints
+* **Technology Flexibility** - Teams enabled to select the best technology for a given problem space, without impacting other teams
+* **Operational Ownership** - Teams responsible for owning and operating the services they're responsible for, with the freedom to deploy small changes frequently
```
Comment (brian-vendia) [inline posts/2022-05-10-improve-your-data-mesh.md:18]:
```suggestion
* **Operational Ownership** - Teams responsible for owning and operating the services, with the freedom to deploy small changes frequently
```

Comment (brian-vendia) [reply posts/2022-05-10-improve-your-data-mesh.md:18]: Awkward wording - repeating `responsible for`.

File: posts/2022-05-10-improve-your-data-mesh.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+
+#### Engaging with Real-Time Partner Data
+
+While quite a bit has been written about data meshes, much less is written about extending the boundaries of the mesh _outside _of a company's four walls.  There's nothing inherent to a decentralized data mesh that would prevent it from spanning organizational boundaries, but doing so places some new requirements on operational systems.  The value of producing data for partners and consuming  data from partners is [not a new or novel idea](https://hbr.org/1990/09/information-partnerships-shared-data-shared-scale).  The continued challenge is to do [so effectively and in real-time](https://www.vendia.net/blog/smart-apis), as if the data really was "shared" among partners and not a (batched) afterthought.
```
Comment (brian-vendia) [inline posts/2022-05-10-improve-your-data-mesh.md:58]:
```suggestion
While quite a bit has been written about data meshes, much less is written about extending the boundaries of the mesh _outside _ of a company's four walls.  There's nothing inherent to a decentralized data mesh that would prevent it from spanning organizational boundaries, but doing so places some new requirements on operational systems.  The value of producing data for partners and consuming  data from partners is [not a new or novel idea](https://hbr.org/1990/09/information-partnerships-shared-data-shared-scale).  The continued challenge is to do [so effectively and in real-time](https://www.vendia.net/blog/smart-apis), as if the data really was "shared" among partners and not a (batched) afterthought.
```

**Review by brian-vendia — CHANGES_REQUESTED** [review]
Super minor changes requested - looks great!

**Review by francine-vendia — APPROVED** [review]

**Review by brian-vendia — APPROVED** [review]
LGTM!

Feel free to ship once tests pass.

**Pull request merged**